.eslintrc.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
     'import/no-extraneous-dependencies': 'off',
     'import/prefer-default-export': 'off',
     'prefer-promise-reject-errors': 'off',
+    'linebreak-style': 0,
```

Comment:
Remove this line.

The correct fix is to set your editor to use LF for line endings. This is standard practice, especially in web.
Also, run `yarn lint` as you develop to find issues. Fix all errors and warnings. Use `yarn lint --fix` to fix the easy things automatically.